morphic.js

```diff
@@ -7818,10 +7824,40 @@ MenuMorph.prototype.select = function (aMenuItem) {
     this.selection = aMenuItem;
 };
 
+MenuMorph.prototype.mouseLeave = function () {
+    if (this.parent instanceof MenuMorph) {
+        // this is a submenu so we vanish when lose focus
+		this.destroy();
```

Comment:
Ah, yes, I agree that parent menus should always be visible. If we need to, we can do something clever like showing menus on the left is necessary. 

Actually...should RTL languages do that anyway? (Do we support RTL at all?)